### PR TITLE
Mobile Nav

### PR DESCRIPTION
**Pull request by kongsally**

### CODE REVIEW

**Comment by kongsally** [comment]
r @thomaslee1911 @nancywong

File: app/assets/styles/app.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Semantic UI breakpoints
 $mobile-breakpoint:        768px;
-$tablet-breakpoint:        992px;
+$tablet-breakpoint:        956px;
```
Comment (sandlerben) [inline app/assets/styles/app.scss]: Why did you change this?

Comment (kongsally) [inline app/assets/styles/app.scss]: That was the point where the last option (log in) was breaking into two lines because there is not enough space. But I think we can revert it back since we merged account into the dashboard

File: app/templates/macros/nav_macros.html [hunk 2]
```diff
@@ -51,7 +70,7 @@
 {# If `count` and `endpoints` are specified, the endpoints will be put into a
  # secondary menu. `count` should be the string (e.g. 'four') number of endpoints. #}
 {% macro desktop_nav(current_user, endpoints=None, count=None) %}
-    <div class="computer tablet only row">
+    <div class="computer only row">
```
Comment (sandlerben) [inline app/templates/macros/nav_macros.html]: What do tablets see then?

Comment (sandlerben) [inline app/templates/macros/nav_macros.html]: Oh jk I see that you moved tablet up

Comment (sandlerben) [inline app/templates/macros/nav_macros.html]: Why did you do that btw?

**Comment by sandlerben** [comment]
Sally looks good except for a few outstanding questions I commented

**Comment by kongsally** [comment]
The questions you had were regarding the nav for tablet and changing the breakpoints, but those are updated now since we have enough space for the tablet and desktop to use the same nav menu.
